dandi/validate.py

```diff
@@ -53,12 +47,11 @@ def validate_bids(
     import bidsschematools
     from bidsschematools.validator import validate_bids as validate_bids_
 
-    if report and not report_path:
-        log_dir = appdirs.user_log_dir("dandi-cli", "dandi")
-        report_path = "{log_dir}/bids-validator-report_{{datetime}}-{{pid}}.log"
-        report_path = report_path.format(
-            log_dir=log_dir,
-        )
+    log_dir = appdirs.user_log_dir("dandi-cli", "dandi")
+    report_path = "{log_dir}/bids-validator-report_{{datetime}}-{{pid}}.log"
+    report_path = report_path.format(
```

Comment:
Do we ever notify the user about the existence of this report?

Reply:
as before I would also argue to make it optional. FWIW we already store the log file for the run, so if desired can log what is desired to be said.

Reply:
@yarikoptic it's a bit more difficult now, since the validation is run by a method of the object. Not sure whether we should be burdening the object with this method parameter. The default DANDI logging does not log the validator report as that's just a dictionary produced and optionally written to a file by an external function. We could disable it entirely, but that means if we have issues, we need to ask people to run the `bst validate` command directly to get the actual report listing all the comparisons.

Reply:
> The default DANDI logging does not log the validator report as that's just a dictionary produced and optionally written to a file by an external function

we get to validation in 2 cases:
- direct use of `dandi validate` -- there we render/output results AFAIK
- optional validation during `upload` -- there I believe we do logging at https://github.com/dandi/dandi-cli/blob/master/dandi/upload.py#L161 

am I missing some other use case where we should log bids-validator output in custom/separate file ?